source/plugins/go/src/utils_windows.go

```diff
@@ -0,0 +1,33 @@
+//go:build windows
+
+package main
+
+import (
+	"context"
+	"syscall"
+	"time"
+
+	"github.com/Microsoft/go-winio"
+)
+
+func CreateWindowsNamedPipesClient(namedPipe string) {
```

Comment:
This should be "CreateWindowsNamedPipeClient" not the "CreateWindowsNamedPipesClient", right?